packages/client/src/pages/Forum/compontents/ForumBox/index.tsx

```diff
@@ -0,0 +1,91 @@
+import {useState} from 'react';
+import {useInput} from 'client/src/hooks/useInput';
+import {ForumColumn} from '../ForumColumn';
+import {ForumHeader} from '../ForumHeader';
+import {Button} from 'client/src/components/Button';
+import styles from './ForumBox.module.scss';
+
+const dataMock = [
+	{
+		id: 1,
+		forums: 'lalal',
+		topics: 58,
+		answers: 77
+	},
+	{
+		id: 2,
+		forums: 'laladdwl',
+		topics: 5,
+		answers: 8
+	},
+	{
+		id: 3,
+		forums: 'PRiVET',
+		topics: 5,
+		answers: 8
+	}
+];
+
+type TDataMock = {
+	id: number;
+	forums: string;
+	topics: number;
+	answers: number;
+};
+
+export const ForumBox = () => {
+
+	const [forumTopics, setFormTopics] = useState(dataMock);
+	const newTopic = useInput('');
+
+	const ForumColumnElements = forumTopics.map((forum: TDataMock) => (
```

Comment:
Думаю уместней будет типизация `useState`:
```suggestion
	const [forumTopics, setFormTopics] = useState<TDataMock[]>(dataMock);
	const newTopic = useInput('');

	const ForumColumnElements = forumTopics.map((forum) => (
```